module/applications/actor/group-sheet.mjs

```diff
@@ -0,0 +1,198 @@
+import ActorMovementConfig from "./movement-config.mjs";
+
+/**
+ * A character sheet for group-type Actors.
+ * The functionality of this sheet is sufficiently different from other Actor types that we extend the base
+ * Foundry VTT ActorSheet instead of the ActorSheet5e abstraction used for character, npc, and vehicle types.
+ */
+export default class GroupActorSheet extends ActorSheet {
+
+  /** @inheritDoc */
+  static get defaultOptions() {
+    return foundry.utils.mergeObject(super.defaultOptions, {
+      classes: ["dnd5e", "sheet", "actor", "group"],
+      template: "systems/dnd5e/templates/actors/group-sheet.hbs",
+      tabs: [{navSelector: ".tabs", contentSelector: ".sheet-body", initial: "members"}],
+      width: 620,
+      height: 620
+    });
+  }
+
+  /* -------------------------------------------- */
+  /*  Context Preparation                         */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async getData(options={}) {
+    const context = super.getData(options);
+
+    // Membership
+    const {sections, stats} = this.#prepareMembers();
+    Object.assign(context, stats);
+    context.sections = sections;
+
+    // Movement
+    context.movement = this.#prepareMovementSpeed();
+
+    // Biography HTML
+    context.descriptionFull = await TextEditor.enrichHTML(this.actor.system.description.full, {
+      secrets: this.actor.isOwner,
+      rollData: context.rollData,
+      async: true,
+      relativeTo: this.actor
+    });
+
+    // Summary tag
+    context.summary = game.i18n.format("DND5E.GroupSummary", {
+      members: [
+        stats.nMembers ? `${stats.nMembers} ${game.i18n.localize("DND5E.GroupMembers")}` : "",
+        stats.nVehicles ? `${stats.nVehicles} ${game.i18n.localize("DND5E.GroupVehicles")}` : ""
+      ].filterJoin(` ${game.i18n.localize("and")} `)
+    })
+    return context;
+  }
+
+  /* -------------------------------------------- */
+
+  /**
+   * Prepare membership data for the sheet.
+   */
+  #prepareMembers() {
+    const stats = {
+      currentHP: 0,
+      maxHP: 0,
+      nMembers: 0,
+      nVehicles: 0
+    }
+    const sections = {
+      character: {label: "Player Characters", members: []},
+      npc: {label: "Non-Player Characters", members: []},
+      vehicle: {label: "Vehicles", members: []}
+    };
+    for ( const member of this.object.system.members ) {
+      const m = {
+        actor: member,
+        id: member.id,
+        name: member.name,
+        img: member.img,
+        hp: {}
+      };
+
+      // HP bar
+      const hp = member.system.attributes.hp;
+      m.hp.current = hp.value + (hp.temp || 0);
+      m.hp.max = hp.max + (hp.tempMax || 0);
+      m.hp.pct = ((m.hp.current / m.hp.max) * 100).toFixed(2);
```

Comment:
Temp HP and temp max HP lead to the bars expanding past the end of the container:

<img width="618" alt="Screenshot 2022-11-19 at 10 49 41" src="https://user-images.githubusercontent.com/19979839/202867042-96851b7b-7bca-4349-98dc-f7c8310ef0ae.png">

```suggestion
      m.hp.max = hp.max + (hp.tempmax || 0);
      m.hp.pct = Math.clamped((m.hp.current / m.hp.max) * 100, 0, 100).toFixed(2);
```

That will fix the immediate problem, but we'll probably want to do a temp HP overlay like on the token bars.